ibm/config.go

```diff
@@ -650,6 +655,10 @@ func (session clientSession) PushServiceV1() (*pushservicev1.PushServiceV1, erro
 	return session.pushServiceClient, session.pushServiceClientErr
 }
 
+func (session clientSession) AppConfigurationV1() (*appconfigurationv1.AppConfigurationV1, error) {
+	return session.appConfigurationClient, session.appConfigurationClientErr
+}
+
```

Comment:
if sess.BluemixSession == nil { in this section please add
session.appConfigurationClientErr = errEmptyBluemixCredentials
}

Reply:
added